mobile/packages/swap/adapters/api/dexhunter/transformers.test.ts

```diff
@@ -941,42 +941,43 @@ describe('toPriceImpact', () => {
-    ${Dex.Wingriders_v1}  | ${Swap.Protocol.Wingriders_v1}
-    ${Dex.Wingriders_v2}  | ${Swap.Protocol.Wingriders_v2}
-    ${Dex.Vyfi_v1}        | ${Swap.Protocol.Vyfi_v1}
-    ${Dex.Sundaeswap_v1}  | ${Swap.Protocol.Sundaeswap_v1}
-    ${Dex.Sundaeswap_v3}  | ${Swap.Protocol.Sundaeswap_v3}
-    ${Dex.Splash_v1}      | ${Swap.Protocol.Splash_v1}
-    ${Dex.Muesliswap_v2}  | ${Swap.Protocol.Muesliswap_v2}
-    ${Dex.Muesliswap_clp} | ${Swap.Protocol.Muesliswap_clp}
-    ${Dex.Unsupported}    | ${Swap.Protocol.Unsupported}
-    ${'new-protocol'}     | ${Swap.Protocol.Unsupported}
+    dex                  | protocol
+    ${Dex.Minswap_v1}    | ${Swap.Protocol.Minswap_v1}
+    ${Dex.Minswap_v2}    | ${Swap.Protocol.Minswap_v2}
+    ${Dex.Wingriders_v1} | ${Swap.Protocol.Wingriders_v1}
+    ${Dex.Wingriders_v2} | ${Swap.Protocol.Wingriders_v2}
+    ${Dex.Vyfi_v1}       | ${Swap.Protocol.Vyfi_v1}
+    ${Dex.Sundaeswap_v1} | ${Swap.Protocol.Sundaeswap_v1}
+    ${Dex.Sundaeswap_v3} | ${Swap.Protocol.Sundaeswap_v3}
+    ${Dex.Splash_v1}     | ${Swap.Protocol.Splash_v1}
+    ${Dex.Unsupported}   | ${Swap.Protocol.Unsupported}
+    ${'new-protocol'}    | ${Swap.Protocol.Unsupported}
   `('should map $dex to $protocol', ({dex, protocol}) => {
     expect(toSwapProtocol(dex)).toBe(protocol)
   })
 })
 
 describe('fromSwapProtocol', () => {
   it.each`
-    protocol                        | dex
-    ${Swap.Protocol.Minswap_v1}     | ${Dex.Minswap_v1}
-    ${Swap.Protocol.Minswap_v2}     | ${Dex.Minswap_v2}
-    ${Swap.Protocol.Wingriders_v1}  | ${Dex.Wingriders_v1}
-    ${Swap.Protocol.Wingriders_v2}  | ${Dex.Wingriders_v2}
-    ${Swap.Protocol.Vyfi_v1}        | ${Dex.Vyfi_v1}
-    ${Swap.Protocol.Sundaeswap_v1}  | ${Dex.Sundaeswap_v1}
-    ${Swap.Protocol.Sundaeswap_v3}  | ${Dex.Sundaeswap_v3}
-    ${Swap.Protocol.Splash_v1}      | ${Dex.Splash_v1}
-    ${Swap.Protocol.Muesliswap_v2}  | ${Dex.Muesliswap_v2}
-    ${Swap.Protocol.Muesliswap_clp} | ${Dex.Muesliswap_clp}
-    ${Swap.Protocol.Teddy_v1}       | ${Dex.Unsupported}
-    ${Swap.Protocol.Minswap_stable} | ${Dex.Unsupported}
-    ${Swap.Protocol.Spectrum_v1}    | ${Dex.Unsupported}
-    ${Swap.Protocol.Unsupported}    | ${Dex.Unsupported}
-    ${'new-protocol'}               | ${Dex.Unsupported}
+    protocol                              | dex
+    ${Swap.Protocol.Minswap_v1}           | ${Dex.Minswap_v1}
+    ${Swap.Protocol.Minswap_v2}           | ${Dex.Minswap_v2}
+    ${Swap.Protocol.Wingriders_v1}        | ${Dex.Wingriders_v1}
+    ${Swap.Protocol.Wingriders_v2}        | ${Dex.Wingriders_v2}
+    ${Swap.Protocol.Vyfi_v1}              | ${Dex.Vyfi_v1}
+    ${Swap.Protocol.Sundaeswap_v1}        | ${Dex.Sundaeswap_v1}
+    ${Swap.Protocol.Sundaeswap_v3}        | ${Dex.Sundaeswap_v3}
+    ${Swap.Protocol.Splash_v1}            | ${Dex.Splash_v1}
+    ${Swap.Protocol.Teddy_v1}             | ${Dex.Unsupported}
+    ${Swap.Protocol.Muesliswap}           | ${Dex.Muesliswap}
```

Comment:
@jorbuedo types were complaining about v2 and clp dunno if they are unsupported now can you triple check